webapp/src/types/api.d.ts

```diff
@@ -118,4 +118,6 @@ export interface CurriculumActivity extends Entity {
 
 interface CalendarEvent extends RBCEvent {
   description: string;
+  //todo
```

Comment:
Only the most necessary of comments should be kept when the pull request is marked as ready for review.